### ISSUE TITLE
The application doesn't work with IP based URL's

### ISSUE THREAD

**Issue by segmentationfaulter** [ushahidi/SMSSync#368]
I was testing SMSSync and I built a web service locally which had a url http://192.168.1.4:3000/ , but SMSSync kept telling me that the endpoint is not valid. Then I deployed the same web-service on heroku and it worked.  Is it a bug or what?

**Comment by rm2k** [comment]
1. Is your phone and web service in the same network?
2. Also, you ought to enter the full URL including the filename. But I believe since it works on heroku it should work locally?

**Comment by segmentationfaulter** [comment]
Yes, phone and web-service were on the same network. EnvayaSMS worked with the same IP based URL but SMSSync did not.
